tests/mocha/test_helpers.js

```diff
@@ -40,38 +40,41 @@ function captureWarnings(innerFunc) {
 }
 
 /**
- * Creates stub for Blockly.utils.genUid that returns the provided id or ids.
- *    Recommended to also assert that the stub is called the expected number of
- *    times.
- * @param {string|!Array<string>} returnIds The return values to use for the
- *    created stub. If a single value is passed, then the stub always returns
- *    that value.
- * @return {!SinonStub} The created stub.
+ * Safely disposes of Blockly workspace, logging any errors.
+ * Assumes that sharedTestSetup has also been called. This should be called
+ * using workspaceTeardown.call(this).
```

Comment:
Here, as well as in `sharedTestSetup` and `sharedTestTeardown`, I'm not sure of a better way to explain how the correct `this` should be bound to the method.